pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -437,3 +441,9 @@ const (
 	// It will be removed by the 'issuing' controller upon completing issuance.
 	CertificateConditionIssuing CertificateConditionType = "Issuing"
 )
+
+// CertificateSecretTemplate defines the default labels and annotations
+// to be copied to the Kubernetes Secret resource named in `CertificateSpec.secretName`.
+type CertificateSecretTemplate struct {
+	Metadata metav1.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
`metav1.ObjectMeta` is problematic to embed and contains a _lot_ of fields that I don't think we want users to be able to set - if this PR is to be accepted, I think limiting this to a subset of what we strictly _do_ want to support is important

Reply:
So instead of using ObjectMeta, what do you think about:

```go
type CertificateSecretTemplate struct {
  Annotations map[string]string `json:"annotations,omitempty" 
  Labels      map[string]string `json:"labels,omitempty"
}
```

Reply:
changed to include only Annotations and Labels for now.

Reply:
@munnerz do you think Annotations and Labels should be enough here?